rules-themed/rules-cat/pom.xml

```diff
@@ -3,11 +3,11 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>com.azure</groupId>
+        <groupId>org.jboss.windup.rules</groupId>
         <artifactId>windup-rulesets-themed</artifactId>
         <version>6.3.0-SNAPSHOT</version>
     </parent>
 
-    <artifactId>windup-rulesets-azure</artifactId>
+    <artifactId>windup-rulesets-cat</artifactId>
```

Comment:
could we just keep azure here?

Reply:
Yes, I agree. Azure is easier to understand for the community

Reply:
Ah OK, that's fine by me then

Reply:
Makes sense to keep the same naming convention.